src/pages/docs/creators/python/examples/tools/converting-docker-image-to-golem-format.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Converting an image
+description: Instructions on how to convert an image from Docker to Golem using gvmkit-build
+type: instructions
+---
+
+# Converting an image
+
+{% alert level="info" %}
+
+These instructions has been designed to work with the following environments:
+
+- OS X 10.14+, Ubuntu 20.04 or Windows
+- Node.js 16.0.0 or above
```

Comment:
I have no idea why this article, which is in the "Python" section, requires node and why it includes Javascript instructions ...

Reply:
tools section is copied from js, will be moved out of the language section to separate menu item, as they are common for both python and js